### PR TITLE
Minor fixes in fhc build docs

### PR DESCRIPTION
**Pull request by mmusil**
@psturc Have a minute for quick review?

### CODE REVIEW

**Comment by fh-automatron** [comment]
Can one of the admins verify this patch?

**Comment by fh-automatron** [comment]
Can one of the admins verify this patch?

**Comment by psturc** [comment]
retest this please

**Comment by fh-automatron** [comment]
### Pull Request Quality Guidelines

As we continue to focus on quality, here are a few things to keep in mind:

* The code changes conform to the contributing & code style guides for this component (if they exist)
* A component owner should review your change (we've tagged these above)
* Automated testing has been added or updated for this change (e.g. UART, Unit/Acceptance tests)
* If updating server component this PR should be deployed to a grid somewhere (via `fhcap-cli`) - please provide a link to this
* Ensure JIRA is up to date before & after merge (e.g. deployment notes, tester notes, release notes, fix version)
* Any SonarQube issues should be addressed where applicable (use your own judgement)

Please ping a component owner (see `fh contrib`) when your builds are complete, changes are deployed and you're ready for a review.
Fhcap PR will be generated automatically.
To deploy your changes to a cluster place comment in this PR with pattern [x][y] where x=deploy and y=clustername you wish to deploy. 
You can also deploy your change manually using fhcap-cli

File: lib/cmd/common/build.js [hunk 1]
```diff
@@ -3,30 +3,31 @@ module.exports = build;
...
-  + i18n._("\nwhere <branch name> is the name of the git branch to build, defaults to 'master'")
-  + i18n._("\nwhere <commit hash> is the full hash of the commit to build, if not the head of the branch. Must be used with <branch name>")
-  + i18n._("\n'keypass' only needed for 'release' builds");
+  + i18n._("\n\n")
+  + i18n._("\n\t<destination>         One of: android, iphone, ipad, ios(for universal binary), blackberry, windowsphone7, windowsphone (windows phone 8).")
+  + i18n._("\n\t<version>             Specific to the destination (e.g. Android version 4.0).")
+  + i18n._("\n\t<config>              Either 'debug' (default), 'distribution', or 'release'.")
+  + i18n._("\n\t<provisioning>        The path to the provisioning profile.")
+  + i18n._("\n\t<cordova_version>     For specifying which version of Cordova to use. Currently supported: either 2.2 or 3.3. Only valid for Android for now.")
+  + i18n._("\n\t<tag name>            The name of a connection tag to be built. Connections tag refers to the cloud app.")
```
Comment (pb82) [inline lib/cmd/common/build.js]: should we also mention here that it has to be in semver format?

File: doc/common/build.md [hunk 2]
```diff
@@ -3,12 +3,15 @@ fhc-build(1) -- Build FeedHenry Applications
...
-      where <cordova_version> is for specifying which version of Cordova to use. Currently supported: either 2.2 or 3.3. Only valid for Android for now.
+
+      fhc build app=<appId> destination=<destination> version=<version> config=<config> keypass=<private-key-password> certpass=<certificate-password> download=<true|false> cordova_version=<cordova-version>
+
+      <destination>         One of: android, iphone, ipad, ios(for universal binary), blackberry, windowsphone7, windowsphone (windows phone 8).
+      <version>             Specific to the destination (e.g. Android version 4.0).
+      <config>              Either 'debug' (default), 'distribution', or 'release'.
+      <provisioning>        The path to the provisioning profile.
+      <cordova_version>     For specifying which version of Cordova to use. Currently supported: either 2.2 or 3.3. Only valid for Android for now.
+      <tag name>            The name of a connection tag to be built. Connections tag refers to the cloud app.")
```
Comment (pb82) [inline doc/common/build.md]: should we also mention here that it has to be in semver format?

File: lib/cmd/common/build.js (hunk 1 above)
Comment (mmusil) [inline lib/cmd/common/build.js]: You are right. It should be there. Maybe forget it while copy pasting.

**Pull request closed without merging**